### PR TITLE
i18n(fr): Update `guides/actions.mdx` from #9781

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/actions.mdx` from #9781

#### Related issues & labels (optional)

- Suggested label: i18n

I'm participating in **Hacktoberfest 2024**!

### CODE REVIEW

File: src/content/docs/fr/guides/actions.mdx [hunk 1]
```diff
@@ -561,7 +561,8 @@ const result = Astro.getActionResult(actions.addToCart);
 ```
 
 :::caution
-Les données d'action sont transmises à l'aide d'un cookie persistant. **Ce cookie n'est pas chiffré.** En général, nous recommandons de renvoyer les informations minimales requises par votre gestionnaire d'actions (`handler`) pour éviter les vulnérabilités et conserver d'autres informations sensibles dans une base de données.
+Les données d'action sont transmises à l'aide d'un cookie persistant. **Ce cookie n'est pas chiffré** et sa taille est limitée à 4 Ko, bien que la limite exacte puisse varier d'un navigateur à l'autre.
+En général, nous recommandons de renvoyer le minimum d'informations nécessaires à votre gestionnaire d'actions (`handler`) pour éviter les vulnérabilités, et de conserver les autres informations sensibles dans une base de données.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/actions.mdx:565]:
```suggestion
Les données d'action sont transmises à l'aide d'un cookie persistant. **Ce cookie n'est pas chiffré** et sa taille est limitée à 4 Ko, bien que la limite exacte puisse varier d'un navigateur à l'autre.

En général, nous recommandons de renvoyer le minimum d'informations nécessaires à votre gestionnaire d'actions (`handler`) pour éviter les vulnérabilités, et de conserver les autres informations sensibles dans une base de données.
```

**Comment by thomasbnt** [comment]
Thanks for the review @dreyfus92

**Review by pioupia — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM!

**Pull request merged**